utils/tex_bot.py

```diff
@@ -109,6 +113,27 @@ async def committee_role(self) -> discord.Role:
 
         return self._committee_role
 
+    @property
+    async def committee_elect_role(self) -> discord.Role:
+        """
+        Shortcut accessor to the committee-elect role.
+
+        The committee elect role is the role held by committee members
```

Comment:
Role name requires capitalisation & speech marks. Fix: `The "Committee-Elect" role is...`